### PR TITLE
update sdk package starter doc

### PR DESCRIPTION
**Pull request by chunyu3**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md [hunk 1]
```diff
@@ -146,7 +157,7 @@ Generate the library source code files to the directory `<sdkPath>/sdk/<service>
   e.g.
 
   ```shell
-  npx cadl compile --emit @azure-tools/cadl-csharp --output-path /home/azure-sdk-for-net/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src .
+  npx cadl compile --emit @azure-tools/cadl-csharp --output-path /home/azure-sdk-for-net/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src main.cadl
```
Comment (lirenhe) [inline doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:160]: will you also specify using client.cadl if it exists?

Comment (chunyu3) [reply doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:160]: added

File: doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ emitters:
   @azure-tools/cadl-csharp emitter options:  
 
 - `namespace` define the client library namespace. e.g. Azure.IoT.DeviceUpdate.
-- `new-project` indicate it is a new sdk project, a project file (.csproj) will be generated.
+- `new-project` indicate if it is a new sdk project and need to generate a project file (.csproj).
 - `clear-output-folder` indicate if you want to clear up the output folder.
```
[the remark this answers is not in the document]
Comment (chunyu3) [reply doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md:122]: added

**Review by lirenhe — APPROVED** [review]

**Pull request merged**